app/src/main/java/org/mozilla/tv/firefox/webrender/cursor/CursorController.kt

```diff
@@ -62,7 +62,9 @@ class CursorController private constructor(
                 else -> Unit
             }
         }, onSelectKey = { event ->
-            viewModel.onSelectKeyEvent(event.action, legacyViewModel.pos)
+            val motionEvent = event.toMotionEvent(legacyViewModel.pos)
```

Comment:
nit: there should be a `MotionEvent.use {}` annotation function

Reply:
Neat.

Reply:
Actually, this class is deleted in #1912 and I'd rather not rebase over this change, so I'm going to leave it as is.  That's good to know though.